CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -   Load PayPal SDK only on a page that has a donation form (#5376)
 -   Credit Card fields tabbing (#5380)
+-   Long text overflowing outside of the container in Donation receipt (#5390)
+-   Legacy form loading spinner (#5397)
```

Comment:
@alaca In the future, please make sure that these changes are added under the `## Unreleased` heading so that I can keep track of which changes have not yet been released. I'll fix this directly on the release branch.

Reply:
@kevinwhoffman I thought I did. I'll pay more attention next time.